README.md

```diff
@@ -81,6 +81,7 @@ To deploy it just run `kubectl apply -f secretdefinition-sample.yaml`
 | `vault.role-id` | `""` | Vault appRole `role_id`. `VAULT_ROLE_ID` environment would take precedence. |
 | `vault.secret-id` | `""` | Vault appRole `secret_id`. `VAULT_SECRET_ID` environment would take precedence. |
 | `vault.engine` | kv2 | Vault secrets engine to use. Only key/value engines supported. Default is kv version 2 |
+| `vault.approle` | approle | Vault approle path |
```

Comment:
I think `vault.approle-path` would be more consistent. What do you think?

Reply:
> I think `vault.approle-path` would be more consistent. What do you think?

yep